pending/service.go

```diff
@@ -65,6 +71,35 @@ func (s service) Create(ctx context.Context, payload documents.UpdatePayload) (d
 	return doc, s.pendingRepo.Create(accID[:], doc.ID(), doc)
 }
 
+// Update updates a pending document from the payload
+func (s service) Update(ctx context.Context, payload documents.UpdatePayload) (documents.Model, error) {
+	accID, err := contextutil.AccountDID(ctx)
+	if err != nil {
+		return nil, contextutil.ErrDIDMissingFromContext
+	}
+
+	m, err := s.pendingRepo.Get(accID[:], payload.DocumentID)
+	if err != nil {
+		return nil, err
+	}
+
+	if m.GetStatus() != documents.Pending {
```

Comment:
this check is will always fail, no ?
since the pending repo will always store the pending documents, WDYT ?

Reply:
right, since the create will take care of the already anchored flow.